### PR TITLE
Adds PencilKit drawing capability

### PR DESCRIPTION
**Pull request by frosty**
This PR adds a 'drawing' capability, using iOS 13's PencilKit framework. This capability allows you to draw onto the selected image using the system-provided tool palette.

On iPhone (and compact widths on iPad), the tool palette is docked to the bottom of the screen and the image should not be obscured by the palette. Also, the docked palette does not contain undo actions, so you should see these in the editor's toolbar.

On iPad (regular widths), the tool palette is not docked, so you can drag it around the screen wherever you like. You should also see the undo and redo actions in the palette.

Simulator Screen Shot - iPad Pro (11-inch) - 2020-05-05 at 15 24 19

When saving, the annotations should be rendered onto the original image. And of course, this only works on iOS 13, as that's when PencilKit was added.

This PR does _not_ include: scrolling or panning the image or canvas while drawing. I haven't quite got that piece worked out yet.

**To test:**

Build and run! Please test on both iPhone and iPad. Testing on a device is preferable, but PencilKit does work in the simulator. Please test:

- [ ] Drawing controls show and work on iPhone and iPad
- [ ] A tall image (a screenshot works well for this) isn't obscured by the tool palette on iPhone
- [ ] You can undo and redo changes as expected using the toolbar controls
- [ ] The canvas resizes as necessary when rotating or resizing the app
- [ ] Tapping Done updates the image with the drawing contents
- [ ] Tests pass

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by frosty** [comment]
cc @bjtitus if you want to take a look, as I know you've used PencilKit before!

**Comment by bjtitus** [comment]
Tried the branch out and it looks great! Thanks for picking this up.

File: Sources/Capabilities/Drawing/MediaEditorAnnotationView.swift [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+///
+@available(iOS 13.0, *)
+class MediaEditorAnnotationView: UIView {
+
+    private let imageView = UIImageView()
+    private let canvasView = PKCanvasView()
+
+    private var bottomConstraint: NSLayoutConstraint!
+
+    weak var undoObserver: MediaEditorAnnotationViewUndoObserver? = nil
```
Comment (leandroalonso) [inline Sources/Capabilities/Drawing/MediaEditorAnnotationView.swift]: Do we need to explicitly assign `nil` to `undoObserver` for any special reason?

File: Sources/Capabilities/Drawing/MediaEditorDrawing.swift [hunk 2]
```diff
@@ -0,0 +1,120 @@
+import UIKit
+import PencilKit
```
Comment (leandroalonso) [inline Sources/Capabilities/Drawing/MediaEditorDrawing.swift]:
```suggestion
```

**Review by leandroalonso — CHANGES_REQUESTED** [review]
Looks great, @frosty.

I just left minor comments. And two more things related to the `README.md`:

 * Can you add this feature in the list?
* Plus update the image of the library including the Drawing icon: 
https://user-images.githubusercontent.com/7040243/81301171-148fb580-904f-11ea-8f7e-00997401cece.PNG

File: Sources/Capabilities/Drawing/MediaEditorAnnotationView.swift (hunk 1 above)
Comment (frosty) [reply Sources/Capabilities/Drawing/MediaEditorAnnotationView.swift]: Nope! I'll remove it :)

**Comment by frosty** [comment]
Thanks @leandroalonso, I pushed some commits to address your comments!

**Review by leandroalonso — APPROVED** [review]

**Comment by leandroalonso** [comment]
🎉 :shipit:

**Pull request merged**